src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,474 @@
+"""

```

Comment:
Could we please change the name of this to `run_medusa_2.py` or something to that effect. We may implement very different pollutant models in the future